lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -43,36 +43,45 @@ define(function(require) {
      * @param  {jQuery} $container  Reportback container object.
      */
     init: function ($container) {
-      var _this = this;
-      
       this.reportbackContainer = $container;
       this.$reportbackWrapper  = this.reportbackContainer.find(".wrapper");
       this.$reportbackForm     = this.reportbackContainer.find("#dosomething-reportback-form");
       this.$uploadButton       = this.reportbackContainer.find(".js-image-upload-beta");
+      this.$captionField       = this.$reportbackForm.find(".form-item-caption");
       this.$initialGallery     = $container.find(".gallery--reportback");
-      
+
       this.itemsPrefetched = $container.data("prefetched");
       this.itemsTotal      = $container.data("total") - this.itemsPrefetched;
       this.nid             = $container.data("nid");
       this.offset          = this.itemsPrefetched;
-      
+
       if (this.itemsPrefetched < this.itemsTotal) {
         this.apiUrl = this.buildApiUrl();
         this.enableViewMore();
         this.enableResponsive();
       }
-      
-      // Initializes image upload modal.
+
+      this.imageUploadInit();
+    },
+
+
+    imageUploadInit: function() {
+      var _this = this;
       var imageUpload = new ImageUploadBeta(this.$uploadButton);
-      
+      var submittedImage = this.$reportbackForm.find(".submitted-image").length === 0 ? false : true;
```

Comment:
@weerd in this case it will not be boolean and may confuse when debugging. But another one ways is using double negation:
```js
var submittedImage = !!this.$reportbackForm.find(".submitted-image").length;
```